_data/instructors.yml

```diff
@@ -18,6 +18,7 @@
 {image: 'michelle-lanz.jpg', name: 'Michelle Lanz',  bio: "is originally from Minnesota and recently moved to the DMV from Switzerland, where she lived for a majority of the last 20 years.  She is currently teaching the Parent Talk conversation course as well as volunteering in various group and individual capacities in the area.  Michelle first earned a B.S. in Physical Therapy from the University of MN and worked in various hospital, rehab and outpatient settings, both in MN and in Switzerland.  She loves to travel and has been fortunate to see many parts of Europe and Africa while living abroad with her husband and three children."},
 {image: 'elizabeth-larson.jpg', name: 'Elizabeth Larson',  bio: "grew up in Michigan and has recently relocated to Bethesda from Frankfurt, Germany, where she lived for 20 years. Elizabeth studied history, Russian Studies and English in Michigan and earned secondary education teaching certification there. She continued her education at Indiana University, where she earned an M.A. in History. She started teaching ESOL to adults in Germany and fell in love with the profession. She's taught all levels of English and especially loves watching students make tangible progress towards their goals. Elizabeth has traveled widely and has lived in Germany, the Republic of Moldova, and Canada. She lives in Bethesda with her German husband and three children."},
+{image: 'ellyn-leverone.jpg', name: 'Ellyn Leverone', bio: "teaches at EF International Language School's New York campus, as well as at English Now!, where she leads online Friday Outings that highlight hidden gems of New York City. Ellyn’s English classes and private tutoring have covered many subjects, including TOEFL preparation, New York on Film, American Youth Culture on Film and Business English. Ellyn received her B.A. from Kenyon College, with double majors in Spanish and Drama, and her CELTA certificate from the University of Cambridge at Teaching House in New York City. Passionate about travel, Ellyn has also been a travel agent for over 20 years, trekking the globe for the benefit of her clients and herself! Her language skills, including fluency in Spanish have fostered her love of teaching in the international community.  She loves to cook cuisines from many countries, and has taught International Cooking classes for children. Ellyn is a devoted choral singer, and loves to incorporate singing and music in her classes. Ellyn lives with her husband Robert outside of New York City and has two grown children."},
```

Comment:
Not specifically related to this, but could we theoretically determine image URLs from names? Just lowercase them, add hyphenation, and stick on `.jpg`? (Some images would need to be converted from PNG to JPG.)

Reply:
It would take more standardization (e.x. La Pierre the name -> lapierre the file) but it seems like it would be possible.

Reply:
@AndrewLester You could just rename the image file as well.